SchlafEin.m

```diff
@@ -374,7 +374,7 @@
     SED.objects.ctrlbuttons(3) = uicontrol('Style','pushbutton', 'FontUnits', 'normalized', 'FontSize', 0.5, 'String','HY', 'TooltipString', 'Hypnogram', ... 
         'Units','normalized', 'Position',[0.94 0.95 0.05 0.05]);
     SED.objects.ctrlbuttons(3).Callback = @SE_ctrlbuttonpush;    
-    SED.objects.ctrlbuttons(4) = uicontrol('Style','togglebutton', 'FontUnits', 'normalized', 'FontSize', 0.5, 'String','L', 'TooltipString', 'Toggle 75�V Line', ... 
+    SED.objects.ctrlbuttons(4) = uicontrol('Style','togglebutton', 'FontUnits', 'normalized', 'FontSize', 0.5, 'String','L', 'TooltipString', 'Toggle 75µV Line', ... 

```

Comment:
in this line nothing changed, I think it's a bug that it's being displayed as "changed".